babel.config.js

```diff
@@ -34,15 +18,15 @@ module.exports = function(api) {
   return {
     presets: [
       isTestEnv && [
-        require('@babel/preset-env').default,
```

Comment:
What's the genesis of these changes?

Reply:
Whuuut. You've got me.  I wonder if I somehow acquired that while I was fixing up the merge from Staging?